src/table/TableBodyWrapper.jsx

```diff
@@ -39,24 +50,33 @@ export default function TableBodyWrapper({ tableData, constraints, selectionsAPI
   return (
     <TableBody>
       {rows.map((row) => (
-        <TableRow hover role="checkbox" tabIndex={-1} key={row.key}>
+        <TableRow
+          hover={selectionsEnabled && hoverEffect}
+          role="checkbox"
+          tabIndex={-1}
+          key={row.key}
+          className={selectionsEnabled && hoverEffect && classes.hoverTableRow}
+        >
           {columns.map((column, i) => {
             const cell = row[column.id];
             const value = cell.qText;
             const CellRenderer = columnRenderers[i];
             return (
-              <CellRenderer
-                className={classes.tableCell}
-                cell={cell}
-                column={column}
-                value={value}
-                key={column.id}
-                align={column.align}
-                selState={selState}
-                selDispatch={selDispatch}
-              >
-                {value}
-              </CellRenderer>
+              CellRenderer && (
+                <CellRenderer
+                  className={classes.tableCell}
+                  cell={cell}
+                  column={column}
+                  value={value}
+                  key={column.id}
+                  align={column.align}
+                  cellLockedImage={cellLockedImage}
```

Comment:
I'm not sure about this. Have you seen any difference in selection performance? Not sure if react handles that, but it would need to pass that image to every cell, every time you make a selection. Did you try mocking the image when testing?